src/app/Fake.IO.FileSystem/Directory.fs

```diff
@@ -0,0 +1,49 @@
+namespace Fake.IO.FileSystem
+
+open System.IO
+
+module Directory =
+
+    /// Creates a directory if it does not exist.
+    let CreateDir path = 
```

Comment:
`Create`?

Reply:
Yep not yet cleaned all modules, just the DirectoryInfo one.